src/planscape/conditions/migrations/0002_get_rast_tile.py

```diff
@@ -0,0 +1,136 @@
+from django.db import migrations
```

Comment:
on my machine, this file is named conditions/migrations/get_rast_tile.py so the diff isn't showing up properly here. is that change intentional?

Reply:
Unfortunately!  Because it's a new version of the function, I had to create a new file for upload (this one is 0002_...). Let's see if I can get the diff to show up.

Reply:
I'm sorry!  I tried to fix the branch so that it shows only the diffs with the original file, but I couldn't get GitHub to see it.  Here is the diff:

https://github.com/OurPlanscape/Planscape/commit/8982eaea823b2c38108ad5f41fd367c1689599f5

Reply:
thank you for getting that to work!